src/main/kotlin/org/rust/ide/actions/RustFmtFileAction.kt

```diff
@@ -0,0 +1,60 @@
+package org.rust.ide.actions
+
+import com.intellij.execution.ExecutionException
+import com.intellij.notification.NotificationType
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.actionSystem.CommonDataKeys
+import com.intellij.openapi.fileEditor.FileDocumentManager
+import com.intellij.openapi.module.ModuleUtilCore
+import com.intellij.openapi.project.DumbAwareAction
+import com.intellij.openapi.vfs.VfsUtil
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.util.PathUtil
+import org.rust.cargo.toolchain.RustToolchain
+import org.rust.ide.notifications.showBalloon
+import org.rust.lang.RustFileType
+
+class RustFmtFileAction : DumbAwareAction() {
+    override fun update(e: AnActionEvent?) {
+        super.update(e);
+        e ?: return
```

Comment:
didn't checkout the original definition of `update` method 😂 , there indeed no `Nullable` annotation.